docs/usage.rst

```diff
@@ -68,7 +93,15 @@ the ``__RESET__`` string can be written to any writable special file to reset it
 
 .. warning:: Only the markdown checkboxes in the ``.cg-rubric.md`` file should be changed to fill in the rubric.
 
+.. note:: In the case the exact format of a special file is not followed and it cannot be easily recovered, the ``__RESET__`` string can be written to any writable special file to reset it to its server state.
+
 Automatic Grading
 ------------------
```

Comment:
Add here:

We recommend doing Automatic Grading with CodeGrade AutoTest, our highly flexible automatic grading framework. It is, however, also possible to grade automatically with the Filesystem.